sunpy/coordinates/sun.py

```diff
@@ -70,6 +70,11 @@ def _angular_radius(sol_radius, distance):
     return Angle(solar_semidiameter_rad.to(u.arcsec))
 
 
+@u.quantity_input
+def _radius(angular_radius: u.arcsec, distance: u.m):
```

Comment:
I suggest being more verbose in the name of this function, e.g., `_physical_radius_from_angular_radius()`

Reply:
👍  - I went with `_radius_from_angular_radius`, but happy to go more explicit again and put `physical` at the front.